scripts/commands/updateApiDocs.ts

```diff
@@ -318,21 +318,29 @@ async function convertHtmlToMarkdown(
 
   for (const result of results) {
     let path = urlToPath(result.url);
-    if (pkg.hasSeparateReleaseNotes && path.endsWith("release-notes.md")) {
-      // Historical versions use the same release notes files as the current API
-      if (pkg.historical) {
-        continue;
-      }
 
+    // Historical versions with a single release notes file should not
+    // modify the current API's file.
```

Comment:
Great comment and code change. This is now much more clear to me! It's "less DRY", but substantially clearer.